sdk/formrecognizer/ai-form-recognizer/samples-dev/analyzeW2TaxForm.ts

```diff
@@ -34,10 +34,9 @@ async function main() {
 
   const poller = await client.beginAnalyzeDocument(PrebuiltModels.TaxUsW2, file);
 
-  const {
-    documents: [result],
-  } = await poller.pollUntilDone();
+  const { documents } = await poller.pollUntilDone();
 
+  const result = documents?.[0];
```

Comment:
It's annoying, but we can't use `?.` in samples just yet. We still have a few months left to support node versions that don't support it.

```suggestion
  const result = documents && documents[0];
```